kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
> I have no problems with adding a case for moved nodes to ezplatformsearch and documenting it.

That would be great, and about the right place to solve it for solrBundle case right?
Should probably also detect if delayed indexing is disabled and log warning/error in all ezplatformsearch methods that call "registerSearchObject" btw.